lib/Server.js

```diff
@@ -302,19 +302,28 @@ function Server(compiler, options) {
 	}, this);
 
 	if(options.https) {
-		// for keep supporting CLI parameters
-		if(typeof options.https === "boolean") {
+		// pfx support
+		if(options.pfx) {
```

Comment:
Why would this need a new option, instead of it being in the `https` option? It's not used that much, so I would prefer to keep it in the `https` object.